### ISSUE TITLE
For November 2017

### ISSUE THREAD

**Issue by neiman30** [bisq-network/compensation#16]
## Basic information

BSQ amount requested: 1000
BSQ address: B1PU7DoYDHWi2impz86Fvu3ZgsREaBnGJBW

## Details

- Audited the Bisq DAO specification (in communication with [Manfred Karrer](https://github.com/ManfredKarrer)). This resulted in a modified version of the compensation request and voting process in the document "06 Bisq DAO - More details and technical specification v0.6".
- Audited the [Payment account age witness](https://github.com/ManfredKarrer/proposals/blob/limits-by-account-age/payment-account-age-witness.adoc) (in communication with [Manfred Karrer](https://github.com/ManfredKarrer)). This resulted in recommendations regarding proof of publication and some either minor issues.

[E: changed request from 10,000 to a 1,000. The initial sum was due to a confusion]

**Comment by cbeams** [comment]
Hi @neiman30, glad to se you've submitted, thanks. As someone unfamiliar with the details of these two units of work, it's not obvious to me that they would be worth 10,000 BSQ (the equivalent of 10,000 USD at our current working valuation of BSQ).

There are a few factors that, without further info, would likely cause me to vote -1 on this request, or at least to abstain from voting on it:

1. The entirety of @ManfredKarrer's work on the payment account age witness feature was encapsulated in his CR last month for 14,000 BSQ; it seems out of proportion that your work auditing this design, however intensive it may have been, would be considered worth many thousands of BSQ by comparison. Again, this is from a distance on my part, and I would have to defer to Manfred's opinion, but the point is that other similarly distant contributors may feel the same way and be similarly uncomfortable voting +1 here.

2. Basically the same commentary about the DAO review. I wasn't previously aware of your work here, and so feel unqualified to judge the value it has added to Bisq. Again, I am not asserting that it hasn't added this value; just pointing out that I am unable to know without further info, data, links to reviews or conversation, etc.

This is interesting input for our process around CRs and voting; what's best in situations like this? To abstain? To vote -1? To discuss like we are here? Unknown, but in any case I think it's important to establish a culture of honesty and transparency about CRs, to call out doubts and concerns like these whenever they come up.

Thanks, and further comments welcome from all.

**Comment by neiman30** [comment]
Hey Chris, 

I didn't mean to ask for 10,000 dollars of course. My work worths a fraction of your and Manfred. I misunderstood the instructions from the Bisq mailing list, and thought that 1 Bsq = 0.1 USD.

In other words, I planned to ask for a 1000 dollars, not for 10,000. I change the request then to a 1000 Bisq.

I assume that only Manfred himself can vouch for my work. I submitted in coordination with him, though we didn't have time to discuss the exact sum. If it is too high, I could lower it. I have little evidence to offer besides the one of Manfred of course.

**Comment by ManfredKarrer** [comment]
Hi @neiman30 
I think the 1000 BSQ is actually too low. Your contribution led to an important improvement on 2 aspects of the DAO model. 

Here is a quick summary of the results from his review and follow up discussions:
1. The compensation request can be entirely represented on the blockchain so no need to deal with the problems from the "eventually consistency" of P2P network data.
2. We changed to blind voting to avoid the "last minute" vote problem.
There was still not the perfect solution found but at least a working one (though not very elegant). But luckily I just found a better elegant solution, which I am happy to discuss with you soon.
All those changes will be reflected in the tech spec paper for the DAO but still not 100% updated.
